src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -80,15 +90,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty(Comment.MESSAGE_KEY, message);
     commentEntity.setProperty(Comment.TIMESTAMP_KEY, timestamp);
     commentEntity.setProperty(Comment.TIME_KEY, time);
+    commentEntity.setProperty("SchoolId", id);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
 
-    response.sendRedirect("/comments.html");
+    String responseLink = String.format("/comments.html?school-id=%d", id);
+    response.sendRedirect(responseLink);
   }
 
   private String getParameter(HttpServletRequest request, String name, String defaultValue) {
     String value = request.getParameter(name);
     return value == null ? defaultValue : value;
   }
+
+  /** Returns the ID of the current school. */
+  private int getId(HttpServletRequest request) {
+    int id = Integer.parseInt(request.getParameter("school-id"));
+    return id;
```

Comment:
These two lines can be simplified to `return Integer.parseInt(request.getParameter("school-id"));`. 
TODO: Also can throw an error or show an error message if the `school-id` param or value is missing? This can be done in the next PR.

Reply:
Yes, i will make note of that for the next PR.